CONTRIBUTING.md

```diff
@@ -157,6 +157,8 @@ Must be one of the following:
 - **refac**: A code change that improves readability or code structure.
   This may incur internal features. Also, this may increase performance, but it's different
   from `perf` type in that performance improvement is not the goal.
+- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+  Just a sorting build targets or header inclusions must be typed as `style`.
```

Comment:
I'm confused reading this, but I assume you're saying:
``` Suggestion
  Commits such as sorting build targets or header inclusions must be typed as `style`.
```
Alternatively, this may be better:
"Other examples include sorting build targets or header inclusions."